docs/redaction.md

```diff
@@ -131,3 +131,7 @@ It's extremely important that path strings do not originate from user input.
 The `fast-redact` module uses a VM context to syntax check the paths, user input
 should never be combined with such an approach. See the [`fast-redact` Caveat](https://github.com/davidmarkclements/fast-redact#caveat)
 and the [`fast-redact` Approach](https://github.com/davidmarkclements/fast-redact#approach) for in-depth information.
+
+## Alternatives
+
+Alternatively to Pino's native redaction, [CabinJS](https://cabinjs.com) may serve your needs out of the box.  CabinJS is a JavaScript logging package and middleware which automatically strips 1600+ sensitive field names, credit card numbers, BasicAuth headers, Social Security Numbers, JSON Web Tokens, API Keys, CSRF Tokens, Stripe Tokens, Passwords, Salts, Hashes, Bank Account Numbers, Bank Routing Numbers, and more.  It also reduces disk storage costs by converting Stream, Buffer, and ArrayBuffer log data into simplified descriptive objects.  Cabin also populates an `app` property in your log metadata which contains your packages name, version, node version, git hash, git tag, environment, hostname, and pid. If you use the widely popular [Passport](https://github.com/jaredhanson/passport) package, then it will even auto-add a user object for you to your log metadata. Lastly, it is a drop-in replacement and alternative to many existing logging solutions, including bunyan, and works great with pino plugged in out of the box.  Read more at <https://cabinjs.com>.
```

Comment:
1. Please line break at ~80 characters.
2. This reads as nothing more than an advertisement. Notice that the rest of this document introduces the feature by showing how to enable it and the effects redaction will have. Reading this text, I still don't know how I would use Cabin with Pino.

Reply:
Ok cool I will patch and ping you back with example integration / code snippets added here.  Great points @jsumners - thank you very much.

Reply:
Please remove the vast majority of the text and add an example. I think the first sentence is ok.

Reply:
This is very cool - can we add a note about performance characteristics of this approach. I’m assuming native redaction (fast-redact) will be significantly higher performance 

In future I’d love to see a way for cabin and fast redact to work together (or at least to implement the same concept on fast redact)